bears/haml/HAMLLintBear.py

```diff
@@ -0,0 +1,368 @@
+            character being excluded.
+        :param multiline_pipe:
+            Check if multiple lines are spanned using multiline pipe (|)
+            syntax or not.
+        :param multiline_script:
+            Check if Ruby scripts are spanned over multiple lines using
+            operators or not.
+        :param object_reference_attributes:
+            Check if object reference syntax is used to set the class/id of an
+            element or not. This syntax should not be used since it makes it
+            difficult to find where a particular class attribute is defined in
+            the code. It also creates an unnecessary coupling by directly
+            associating class names to the objects passed to it, making
+            refactoring models affect the views.
+        :param repeat_id:
+            Check whether IDs are unique or not. Repeating an ID is an error
+            in the HTML specification.
+        :param repeat_id_severity:
+            Set the severity level of the linter in checking whether the IDs
+            are unique or not.
+        :param rubo_cop:
+            Use this rule to enable the linter to run RuboCop over the Ruby
+            code in the templates.
+        :param rubo_cop_ignored_cops:
+            Specify cops which are to be ignored while running RuboCop over
+            the Ruby code in the templates.
+        :param ruby_comments:
+            Check if HAML's built-in comments are preferred over ad hoc Ruby
+            comments.
+        :param space_before_script:
+            Check if Ruby script indicators are separated from code with a
+            single space or not.
+        :param space_inside_hash_attributes:
+            Check if the style of hash attributes is one of the two possible
+            preferred styles or not.
+        :param space_inside_hash_attributes_style:
+            Set the preferred style of hash attributes.
+        :param tag_name:
+            Check if the tag names contain uppercase letters or not. Tag names
+            should not contain uppercase letters.
+        :param trailing_whitespace:
+            Check whether trailing whitespace linter is enabled or not.
+        :param unnecessary_interpolation:
+            Check if there is unnecessary interpolation for inline tag content
+            or not. Unnecessary interpolation must be avoided.
+        :param unnecessary_string_output:
+            Check if string expressions are being outputted in Ruby, when
+            static text will suffice. HAML gracefully handles string
+            interpolation in static text, so Ruby strings are not required in
+            order to use interpolation.
+        :param view_length:
+            Check if large views are split into separate partials.
+        :param max_view_length:
+            Set maximum length of template views.
+        :param hamllint_config:
+            Path to a custom configuration file.
```

Comment:
you have to only document in `create_arguments` once, this will be overriden by the docs from there anway ;)
(so just omit `:param hamllint_config:`)